pkg/maintainer/wallet/deposit_sweep.go

```diff
@@ -172,6 +159,20 @@ func FindDeposits(
 			continue
 		}
 
+		var scriptType bitcoin.ScriptType
```

Comment:
If there is a `GetTransaction` error `scriptType` will not be assigned any value, is it expected?
Shouldn't we call `continue` for deposits with `GetTransaction` errors?

Reply:
It is expected. We may want to show that transaction for `list-deposits` CLI command anyway. Useful if we want to determine which deposit transactions are actually fakes.